src/libraries/System.Reflection.Emit/tests/TypeBuilder/TypeBuilderCreateType.cs

```diff
@@ -40,7 +40,6 @@ public void CreateType(TypeAttributes attributes)
         [ActiveIssue("https://github.com/dotnet/runtime/issues/2389", TestRuntimes.Mono)]
```

Comment:
Do you need to fix https://github.com/dotnet/runtime/blob/f4bc4688becd0a576d5a39da32dabe17fe5fcf35/src/libraries/System.Reflection.Emit/src/System/Reflection/Emit/TypeBuilderImpl.cs#L518 as well?